android_app/app/src/main/res/values/strings.xml

```diff
@@ -153,34 +153,34 @@
     <string name="label_backup">Backup</string>
     <string name="label_export_dir">Export dir</string>
     <string name="label_not_found">not found</string>
-    <string name="label_ignoreOutOfRange">Ignore data that are out of range</string>
+    <string name="label_ignoreOutOfRange">Ignore out of range data</string>
     <string name="label_initial_weight">Initial weight</string>
     <string name="label_regression_line">Regression weight line</string>
     <string name="label_regression_line_degree">Regression polynom degree</string>
     <string name="label_goal_line">Goal line</string>
     <string name="label_help">Help</string>
 
     <string name="label_feedback_message_enjoying">Enjoying openScale?</string>
-    <string name="label_feedback_message_rate_app">How about a rating on GooglePlay or on GitHub, then?</string>
-    <string name="label_feedback_message_issue">Would you mind giving us some feedback?</string>
-    <string name="label_feedback_message_yes">Yes!</string>
+    <string name="label_feedback_message_rate_app">How about a rating on Google Play or GitHub?</string>
+    <string name="label_feedback_message_issue">Would you mind providing some feedback?</string>
+    <string name="label_feedback_message_yes">Yes</string>
     <string name="label_feedback_message_no">Not really</string>
-    <string name="label_feedback_message_positive">Ok, sure</string>
-    <string name="label_feedback_message_negative">No, thanks</string>
+    <string name="label_feedback_message_positive">OK</string>
+    <string name="label_feedback_message_negative">No</string>
 
-    <string name="error_max_scale_users">Maximum number of concurrent scale users reached</string>
+    <string name="error_max_scale_users">Max. number of concurrent scale users reached</string>
     <string name="info_step_on_scale">Please step barefoot on the scale for reference measurements</string>
     <string name="info_measuring">Measuring weight: %.2f</string>
 
-    <string name="customactivityoncrash_error_activity_error_occurred_explanation">An unexpected error occurred.\n\nPlease create a new issue including the detailed error details on\nhttps://github.com/oliexdev/openScale/issues</string>
```

Comment:
The "error details" part was a reference to the other string included in the crash screen to indicate that that information should be included.

Reply:
I would think that is implied, but i can change it to that effect.